tests/test_dynamodb.py

```diff
@@ -0,0 +1,125 @@
+from pytest_dbfixtures import factories
+
+dynamodb_proc = factories.dynamodb_proc(
+    jar_path='/tmp/dynamodb/DynamoDBLocal.jar'
```

Comment:
I cannot run these tests without doing some custom undocumented setup, but it's ok, since nearly all other tests also fail by default.